lib/services/address/index.js

```diff
@@ -698,6 +732,26 @@ AddressService.prototype._decodeInputValueMap = function(buffer) {
   };
 };
 
+AddressService.prototype._getAddressInfo = function(addressStr) {
+  var addrObj = bitcore.Address(addressStr);
+  var hashbuffer = addrObj.hashBuffer;
+
+  // Find our internal encoding for this hash type
+  var hashTypes = Object.keys(AddressService.HASH_TYPES_READABLE);
```

Comment:
Any reason we shouldn't add another constant for this, e.g.:
```js
AddressService.HASH_TYPES_MAP = {
  'pubkeyhash': AddressService.HASH_TYPES.PUBKEY,
  'scripthash': AddressService.HASH_TYPES.REDEEMSCRIPT
};
```
So this can just be:
```js
var hashTypeBuffer = AddressService.HASH_TYPES_MAP[addrObj.type];
```